interfaces/Portalicious/src/app/components/page-layout/components/footer/footer.component.html

```diff
@@ -64,8 +85,14 @@ <h3 class="mb-4">Helpful links</h3>
       </ul>
     </div>
     <div>
-      <h3 class="mb-4">Questions or feedback? Contact us</h3>
+      <h3
+        i18n
+        class="mb-4"
+      >
+        Questions or feedback? Contact us
+      </h3>
       <a
+        i18n
         href="mailto:support@510.global"
         class="p-button p-button-rounded p-button-outlined p-button-secondary [&:not(:hover)]:text-white"
         >support&#64;510.global</a
```

Comment:
If we're putting the email-address in plain-text in the `href`, then there is no point in obfuscating it here, I'd think.